tests/kafkatest/tests/core/replica_scale_test.py

```diff
@@ -48,8 +48,28 @@ def teardown(self):
             self.zk.stop()
 
     @cluster(num_nodes=12)
-    @matrix(topic_count=[50], partition_count=[34], replication_factor=[3], metadata_quorum=quorum.all_non_upgrade)
-    def test_produce_consume(self, topic_count, partition_count, replication_factor, metadata_quorum=quorum.zk):
+    @matrix(
+        topic_count=[50],
+        partition_count=[34],
+        replication_factor=[3],
+        metadata_quorum=[quorum.zk],
+        use_new_coordinator=[False]
+    )
+    @matrix(
+        topic_count=[50],
+        partition_count=[34],
+        replication_factor=[3],
+        metadata_quorum=[quorum.isolated_kraft],
+        use_new_coordinator=[True, False]
+    )
+    def test_produce_consume(
```

Comment:
nit: I won't think that we use this style for format method definition in python.